src/widgets/JSONForm/JSONForm.js

```diff
@@ -177,6 +177,7 @@ export const JSONFormComponent = React.forwardRef(
       onSubmit,
       surveySchema,
       liveValidate,
+      disabled,
```

Comment:
Since we're only adding view functionality here, had to add ability to disable the form (otherwise it's confusing as it looks editable but really isn't)

Reply:
ah. I noticed that the Checkboxes component still allows editing - I think it will need to implement `disabled` soz

Reply:
Oops - I didn't actually try tapping the text to enable/disable! Fixed: https://github.com/openmsupply/mobile/pull/4668/commits/7ad5a940af2926e03d27932f3c53cf0959b98a54